React/Views/RefreshControl/RCTRefreshControl.m

```diff
@@ -120,6 +122,19 @@ - (void)endRefreshingProgrammatically
   }
 }
 
+- (void)applyProgressViewOffset
+{
+  // progressViewOffset must be converted from the ScrollView parent's coordinate space to
+  // the coordinate space of the RefreshControl. This ensures that the control respects any
+  // offset in the view hierarchy, and that progressViewOffset is not inadvertently applied
+  // multiple times.
+  UIView *scrollView = self.superview;
+  UIView *target = scrollView == nil ? nil : scrollView.superview;
```

Comment:
Just a tip about obj-c :) Btw you can just call `scrollView.superview`. In case scrollView is nil, nil is returned from the entire expression. No need to handle nil case explicitly. 

I'll change it.

Reply:
Ohhh. You're right, I completely forgot that dot syntax is just sending a message to a property getter here, isn't it? I'll try to remember that for next time. Thanks for the tip (and the correction)!

Reply:
Sending a message to nil does nothing in Obj-c. It is a feature, not a bug :) 

I'll merge this on Monday.

Thank you.